src/Reflection/ReflectionValue.php

```diff
@@ -74,40 +75,40 @@ class ReflectionValue implements ReferenceCountedInterface
     use ReferenceCountedTrait;
 
     /* regular data types */
-    public const IS_UNDEF        = 0;
-    public const IS_NULL         = 1;
-    public const IS_FALSE        = 2;
-    public const IS_TRUE         = 3;
-    public const IS_LONG         = 4;
-    public const IS_DOUBLE       = 5;
-    public const IS_STRING       = 6;
-    public const IS_ARRAY        = 7;
-    public const IS_OBJECT       = 8;
-    public const IS_RESOURCE     = 9;
-    public const IS_REFERENCE    = 10;
-    public const IS_CONSTANT_AST = 11; /* constant expressions */
+    public const IS_UNDEF        = Defines::IS_UNDEF;
+    public const IS_NULL         = Defines::IS_NULL;
+    public const IS_FALSE        = Defines::IS_FALSE;
+    public const IS_TRUE         = Defines::IS_TRUE;
+    public const IS_LONG         = Defines::IS_LONG;
+    public const IS_DOUBLE       = Defines::IS_DOUBLE;
+    public const IS_STRING       = Defines::IS_STRING;
+    public const IS_ARRAY        = Defines::IS_ARRAY;
+    public const IS_OBJECT       = Defines::IS_OBJECT;
+    public const IS_RESOURCE     = Defines::IS_RESOURCE;
+    public const IS_REFERENCE    = Defines::IS_REFERENCE;
+    public const IS_CONSTANT_AST = Defines::IS_CONSTANT_AST;
 
     /**
      * Fake types used only for type hinting.
      * These are allowed to overlap with the types below.
      */
-    public const IS_CALLABLE       = 12;
-    public const IS_ITERABLE       = 13;
-    public const IS_VOID           = 14;
-    public const IS_STATIC         = 15;
-    public const IS_MIXED          = 16;
+    public const IS_CALLABLE       = Defines::IS_CALLABLE;
+    public const IS_ITERABLE       = Defines::IS_ITERABLE;
+    public const IS_VOID           = Defines::IS_VOID;
+//    public const IS_STATIC         = Defines::IS_STATIC;
```

Comment:
These symbols were only added in 8.0 - removed to get 7.4 compatibility but they could be re-added if needed.